ui/src/kapacitor/components/RuleHeaderEdit.js

```diff
@@ -24,18 +24,16 @@ const RuleHeaderEdit = ({
           className="page-header__title page-header--editable kapacitor-theme"
           onClick={onToggleEdit}
           data-for="rename-kapacitor-tooltip"
-          data-tip="Click to Rename"
+          data-tip={'<p>Click to Rename</p>'}
         >
```

Comment:
This is a string so the `data-tip` prop curly braces should be unecessary

Reply:
Done!